lance/lance-rest-server/src/test/java/org/apache/gravitino/lance/integration/test/LanceRESTServiceIT.java

```diff
@@ -0,0 +1,394 @@
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.lancedb.lance.namespace.LanceNamespace;
+import com.lancedb.lance.namespace.LanceNamespaceException;
+import com.lancedb.lance.namespace.LanceNamespaces;
+import com.lancedb.lance.namespace.model.CreateNamespaceRequest;
+import com.lancedb.lance.namespace.model.CreateNamespaceResponse;
+import com.lancedb.lance.namespace.model.DescribeNamespaceRequest;
+import com.lancedb.lance.namespace.model.DescribeNamespaceResponse;
+import com.lancedb.lance.namespace.model.DropNamespaceRequest;
+import com.lancedb.lance.namespace.model.DropNamespaceResponse;
+import com.lancedb.lance.namespace.model.ListNamespacesRequest;
+import com.lancedb.lance.namespace.model.ListNamespacesResponse;
+import com.lancedb.lance.namespace.model.NamespaceExistsRequest;
+import com.lancedb.lance.namespace.rest.RestNamespaceConfig;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.gravitino.Catalog;
+import org.apache.gravitino.NameIdentifier;
+import org.apache.gravitino.Schema;
+import org.apache.gravitino.client.GravitinoMetalake;
+import org.apache.gravitino.integration.test.util.BaseIT;
+import org.apache.gravitino.integration.test.util.GravitinoITUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class LanceRESTServiceIT extends BaseIT {
+
+  private GravitinoMetalake metalake;
+  private Map<String, String> properties =
+      new HashMap<>() {
+        {
+          put("key1", "value1");
+        }
+      };
+  private final BufferAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+  private LanceNamespace ns;
+  private Path tempDir;
+
+  @BeforeAll
+  public void startIntegrationTest() throws Exception {
+    super.ignoreLanceAuxRestService = false;
+    super.startIntegrationTest();
+    this.metalake = createMetalake(getLanceRESTServerMetalakeName());
+
+    HashMap<String, String> props = Maps.newHashMap();
+    props.put(RestNamespaceConfig.URI, getLanceRestServiceUrl());
+    props.put(RestNamespaceConfig.DELIMITER, RestNamespaceConfig.DELIMITER_DEFAULT);
+    this.ns = LanceNamespaces.connect("rest", props, null, allocator);
+
+    this.tempDir = Files.createTempDirectory("test_lance_rest_service_it_");
+    // deleteOnExit
+    tempDir.toFile().deleteOnExit();
+  }
+
+  @AfterAll
+  public void clean() {
+    client.dropMetalake(getLanceRESTServerMetalakeName(), true);
+    tempDir.toFile().deleteOnExit();
+  }
+
+  @AfterEach
+  public void clearMetalake() {
+    Arrays.stream(metalake.listCatalogs()).forEach(c -> metalake.dropCatalog(c, true));
+    tempDir.toFile().deleteOnExit();
```

Comment:
You have done it in L82, it seems that this is unnecessary.

Reply:
ok. keep in afterAll only